pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -442,8 +466,25 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication, ReentrancyGuard {
             return account == address(_authorizerAdaptorEntrypoint);
         }
 
+        // Actions with no delay can only be performed by accounts that have the associated permission.
+        // Actions with a non-zero delay however cannot be performed by permissioned accounts: they can only be made by
+        // the TimelockAuthorizerExecutionHelper, which works alongisde the TimelockAuthorizer itself to ensure that
+        // said executions have been properly scheduled in advance by an authorized party via the `schedule` function.
```

Comment:
```suggestion
        // executions have been properly scheduled in advance by an authorized party via the `schedule` function.
```
Not technically wrong, but using "said" like that is most common as "legalese" in court documents. I'd try to limit it. Can say "aforementioned" if necessary to mean the same thing